### PR TITLE
Fix links for Block, Chunk, ExecutionOutcome, Transaction, Shard, Receipt

### PR DESCRIPTION
**Pull request by PolyProgrammist**

### CODE REVIEW

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 1]
```diff
@@ -33,27 +33,27 @@ In fact, any blockchain flow can be represented as an infinity timeline that has
 And a block appears on that timeline with some interval. Each of these blocks holds the information about the previous one, thus, creating a *chain of blocks*.
 
 
-NEAR Protocol has a sharded nature, meaning that more than one parallel network, called a [Shard](https://near-indexers.io/docs/data-flow-and-structures/structures/shard), can be live at any moment. And each Shard produces a chunk of a block at a given interval. A Block in NEAR Blockchain is a collection of the block chunks from all shards. Block chunk is shortened to [Chunk](https://near-indexers.io/docs/data-flow-and-structures/structures/chunk) in NEAR Protocol documentation.
+NEAR Protocol has a sharded nature, meaning that more than one parallel network, called a [Shard](https://docs.near.org/develop/lake/structures/shard), can be live at any moment. And each Shard produces a chunk of a block at a given interval. A Block in NEAR Blockchain is a collection of the block chunks from all shards. Block chunk is shortened to [Chunk](https://near-indexers.io/docs/data-flow-and-structures/structures/chunk) in NEAR Protocol documentation.
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:36]:
```suggestion
NEAR Protocol has a sharded nature, meaning that more than one parallel network, called a [Shard](https://docs.near.org/develop/lake/structures/shard), can be live at any moment. And each Shard produces a chunk of a block at a given interval. A Block in NEAR Blockchain is a collection of the block chunks from all shards. Block chunk is shortened to [Chunk](https://docs.near.org/develop/lake/structures/chunk) in NEAR Protocol documentation.
```

File: docs/1.concepts/data-flow/token-transfer-flow.md [hunk 2]
```diff
@@ -69,13 +69,13 @@ This is it. Tokens has been transferred from the account on one Shard to the acc
...
+Let's have a look at the example where both accounts are on the same [Shard](https://docs.near.org/develop/lake/structures/shard). The process is the same as in the previous example, except there are no Receipts moving from one Shard to another.
 
-A [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://near-indexers.io/docs/data-flow-and-structures/structures/execution_outcome) is the result of converting the transaction into a [Receipt](https://near-indexers.io/docs/data-flow-and-structures/structures/receipt).
+A [Transaction](https://docs.near.org/concepts/protocol/transactions) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) is the result of converting the transaction into a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```
Comment (bucanero) [inline docs/1.concepts/data-flow/token-transfer-flow.md:74]:
```suggestion
A [Transaction](https://docs.near.org/develop/lake/structures/transaction) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) is the result of converting the transaction into a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 3]
```diff
@@ -33,27 +33,27 @@ In fact, any blockchain flow can be represented as an infinity timeline that has
...
 
 Returning to the data flow itself. The best we can do is to imagine tracks like we could see in audio/video editing apps. Each Shard has its own set of tracks. The top track is for Chunks. They appear no matter what at a given interval, for NEAR Blockchain the interval is about one second. Chunks are being produced even if nothing is happening on the blockchain.
 
 ![Timeline as tracks](/docs/flow/02-tracks.png)
 
-But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well known method to trigger any changes is to send a [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) to the blockchain with instructions of what we want to change and who has requested those changes.
+But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well known method to trigger any changes is to send a [Transaction](https://docs.near.org/concepts/protocol/transactions) to the blockchain with instructions of what we want to change and who has requested those changes.
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:42]:
```suggestion
But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well-known method to trigger any changes is to send a [Transaction](https://docs.near.org/develop/lake/structures/protocol/transaction) to the blockchain with instructions of what we want to change and who has requested those changes.
```

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 4]
```diff
@@ -33,27 +33,27 @@ In fact, any blockchain flow can be represented as an infinity timeline that has
...
 
-A transaction needs to be constructed, signed and sent to the blockchain. Once it is executed we expect to have a result - [ExecutionOutcome](https://near-indexers.io/docs/data-flow-and-structures/structures/execution_outcome). Oh, that is simple. But it’s not really accurate for NEAR Blockchain.
+A transaction needs to be constructed, signed and sent to the blockchain. Once it is executed we expect to have a result - [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome). Oh, that is simple. But it’s not really accurate for NEAR Blockchain.
 
 ![Transaction execution](/docs/flow/03-tx-outcome-receipt.png)
 
-In the beginning there is a [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction), it contains instructions we want to be executed on the blockchain. Transaction is sent to NEAR Blockchain.
+In the beginning there is a [Transaction](https://docs.near.org/concepts/protocol/transactions), it contains instructions we want to be executed on the blockchain. Transaction is sent to NEAR Blockchain.
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:48]:
```suggestion
In the beginning there is a [Transaction](https://docs.near.org/develop/lake/structures/transaction), it contains instructions we want to be executed on the blockchain. Transaction is sent to NEAR Blockchain.
```

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 5]
```diff
@@ -33,27 +33,27 @@ In fact, any blockchain flow can be represented as an infinity timeline that has
...
+Assuming we have two accounts living on different [Shards](https://docs.near.org/develop/lake/structures/shard) **alice.near** and **bob.near**. **alice.near** creates a [Transaction](https://docs.near.org/concepts/protocol/transactions) to send a few tokens to **bob.near**. The [Transaction](https://docs.near.org/concepts/protocol/transactions) is immediately executed and the [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) for the [Transaction](https://docs.near.org/concepts/protocol/transactions) is always a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:54]:
```suggestion
Assuming we have two accounts living on different [Shards](https://docs.near.org/develop/lake/structures/shard) **alice.near** and **bob.near**. **alice.near** creates a [Transaction](https://docs.near.org/develop/lake/structures/transaction) to send a few tokens to **bob.near**. The [Transaction](https://docs.near.org/develop/lake/structures/transaction) is immediately executed and the [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) for the [Transaction](https://docs.near.org/develop/lake/structures/transaction) is always a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 6]
```diff
@@ -69,8 +69,8 @@ So the final scheme will look like:
 
 ## Summary
 
-We have learnt the main principles of how data is flowing in NEAR Protocol. We have found out that [Transactions](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) execute and the [ExecutionOutcome](https://near-indexers.io/docs/data-flow-and-structures/structures/execution_outcome) of the Transaction is always a [Receipt](https://near-indexers.io/docs/data-flow-and-structures/structures/receipt).
+We have learnt the main principles of how data is flowing in NEAR Protocol. We have found out that [Transactions](https://docs.near.org/concepts/protocol/transactions) execute and the [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) of the Transaction is always a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:72]:
```suggestion
We have learnt the main principles of how data is flowing in NEAR Protocol. We have found out that [Transactions](https://docs.near.org/develop/lake/structures/transaction) execute and the [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) of the Transaction is always a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```

File: docs/1.concepts/data-flow/token-transfer-flow.md [hunk 7]
```diff
@@ -41,24 +41,24 @@ OK, this should be enough for introduction, now let's move on to examples.
...
+Assume we have two accounts **alice.near** and **bob.near**. They belong to different [Shards](https://docs.near.org/develop/lake/structures/shard). **alice.near** sends a few tokens to **bob.near**.
 
-A [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://near-indexers.io/docs/data-flow-and-structures/structures/execution_outcome) is the output or result from converting the transaction into a [Receipt](https://near-indexers.io/docs/data-flow-and-structures/structures/receipt).
+A [Transaction](https://docs.near.org/concepts/protocol/transactions) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) is the output or result from converting the transaction into a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```
Comment (bucanero) [inline docs/1.concepts/data-flow/token-transfer-flow.md:46]:
```suggestion
A [Transaction](https://docs.near.org/develop/lake/structures/transaction) signed by **alice.near** is sent to the network. It is immediately executed, [ExecutionOutcome](https://docs.near.org/develop/lake/structures/execution-outcome) is the output or result from converting the transaction into a [Receipt](https://docs.near.org/develop/lake/structures/receipt).
```

File: docs/1.concepts/data-flow/token-transfer-flow.md [hunk 8]
```diff
@@ -41,24 +41,24 @@ OK, this should be enough for introduction, now let's move on to examples.
...
 
 ![Transaction execution](/docs/flow/03-tx-outcome-receipt.png)
 
-During the above process **alice.near**, the sender, was charged a fee (gas). The [Receipt](https://near-indexers.io/docs/data-flow-and-structures/structures/receipt) created as result of the [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) follows these rules:
+During the above process **alice.near**, the sender, was charged a fee (gas). The [Receipt](https://docs.near.org/develop/lake/structures/receipt) created as result of the [Transaction](https://docs.near.org/concepts/protocol/transactions) follows these rules:
```
Comment (bucanero) [inline docs/1.concepts/data-flow/token-transfer-flow.md:50]:
```suggestion
During the above process **alice.near**, the sender, was charged a fee (gas). The [Receipt](https://docs.near.org/develop/lake/structures/receipt) created as result of the [Transaction](https://docs.near.org/develop/lake/structures/transaction) follows these rules:
```

**Review by bucanero — APPROVED** [review]

File: docs/1.concepts/data-flow/near-data-flow.md [hunk 9]
```diff
@@ -33,27 +33,27 @@ In fact, any blockchain flow can be represented as an infinity timeline that has
...
 
 Returning to the data flow itself. The best we can do is to imagine tracks like we could see in audio/video editing apps. Each Shard has its own set of tracks. The top track is for Chunks. They appear no matter what at a given interval, for NEAR Blockchain the interval is about one second. Chunks are being produced even if nothing is happening on the blockchain.
 
 ![Timeline as tracks](/docs/flow/02-tracks.png)
 
-But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well known method to trigger any changes is to send a [Transaction](https://near-indexers.io/docs/data-flow-and-structures/structures/transaction) to the blockchain with instructions of what we want to change and who has requested those changes.
+But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well-known method to trigger any changes is to send a [Transaction](https://docs.near.org/develop/lake/structures/protocol/transaction) to the blockchain with instructions of what we want to change and who has requested those changes.
```
Comment (bucanero) [inline docs/1.concepts/data-flow/near-data-flow.md:42]:
```suggestion
But what do we mean by saying something is happening? We mean that something has triggered some changes in the blockchain. And the most well-known method to trigger any changes is to send a [Transaction](https://docs.near.org/develop/lake/structures/transaction) to the blockchain with instructions of what we want to change and who has requested those changes.
```

**Comment by PolyProgrammist** [comment]
Thanks @bucanero

**Pull request merged**